datasets/_global/shu_uyghur_companies/shu_uyghur_companies.yml

```diff
@@ -0,0 +1,81 @@
+title: Helena Kennedy Centre Companies Operating in the Uyghur Region
+entry_point: crawler.py
+prefix: shu-xj
+coverage:
+  frequency: daily
+  start: 2024-12-12
+load_db_uri: ${OPENSANCTIONS_DATABASE_URI}
+ci_test: false
+summary: >-
+  Over 50,000 companies operating across various sectors in the Uyghur Region.  
+description: |
+  This dataset highlights the corporate landscape of the Uyghur Region, including 
+  companies involved in manufacturing, export, professional services, construction, 
+  telecommunications, and retail. It provides names, addresses, and sectors in both 
+  Chinese and English.
+
+  Its purpose is to provide a better understanding of the economic landscape in the 
+  Uyghur Region amid ongoing human rights concerns. This list represents publicly 
+  available information and is not an original research product of the authors or 
+  of Sheffield Hallam University.
+
+  **Note:** The list is drawn from a human resources website, accessed via the internet in 
+  December 2021. This list is not exhaustive, nor does it provide data on companies that were 
+  registered, de-registered, or otherwise changed status after that time.
+  English translation is machine-generated and may not be accurate.
+  [Source: Helena Kennedy Centre for International Justice](https://www.shu.ac.uk/helena-kennedy-centre-international-justice/research-and-projects/all-projects/useful-resources)
```

Comment:
I wonder if we should assert a hash and indicate the static nature of this website in the metadata. Then the hash assertion will tell is if they do another set of research and update it letting us know to update the metadata.